package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "marko-tester",
-  "version": "6.32.4",
+  "version": "6.33.0",
```

Comment:
Don't do manual version update, it's automated.

Reply:
ok